Wrong imports path in @spartacus/user library

```
node_modules/@spartacus/user/profile/core/facade/user-email.service.d.ts:11:37 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.
    11     protected updateCommand: import("../../../../../dist/core/spartacus-core").Command<{
                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    node_modules/@spartacus/user/profile/core/facade/user-password.service.d.ts:10:37 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.
    10     protected updateCommand: import("../../../../../dist/core/spartacus-core").Command<{
                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    node_modules/@spartacus/user/profile/core/facade/user-password.service.d.ts:14:36 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.
    14     protected resetCommand: import("../../../../../dist/core/spartacus-core").Command<{
                                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    node_modules/@spartacus/user/profile/core/facade/user-password.service.d.ts:18:57 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.
    18     protected requestForgotPasswordEmailCommand: import("../../../../../dist/core/spartacus-core").Command<{
                                                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    node_modules/@spartacus/user/profile/core/facade/user-profile.service.d.ts:15:37 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.
    15     protected updateCommand: import("../../../../../dist/core/spartacus-core").Command<{
                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    node_modules/@spartacus/user/profile/core/facade/user-profile.service.d.ts:18:36 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.
    18     protected closeCommand: import("../../../../../dist/core/spartacus-core").Command<undefined, unknown>;
                                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    node_modules/@spartacus/user/profile/core/facade/user-profile.service.d.ts:19:34 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.
    19     protected titleQuery: import("../../../../../dist/core/spartacus-core").Query<Title[], []>;
                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    node_modules/@spartacus/user/profile/core/facade/user-register.service.d.ts:15:39 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.
    15     protected registerCommand: import("../../../../../dist/core/spartacus-core").Command<{
                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    node_modules/@spartacus/user/profile/core/facade/user-register.service.d.ts:18:44 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.
    18     protected registerGuestCommand: import("../../../../../dist/core/spartacus-core").Command<{
    ```

--- a/feature-libs/user/account/core/facade/user-account.service.ts
+++ b/feature-libs/user/account/core/facade/user-account.service.ts
@@ -8,6 +8,7 @@ import {
 import {
   LoginEvent,
   LogoutEvent,
+  Query,
   QueryService,
   UserIdService,
 } from '@spartacus/core';
@@ -16,7 +17,7 @@ import { UserAccountConnector } from '../connectors/user-account.connector';
 
 @Injectable()
 export class UserAccountService implements UserAccountFacade {
-  protected userQuery = this.query.create(
+  protected userQuery: Query<User> = this.query.create(
     () =>
       this.userIdService
         .takeUserId(true)

--- a/feature-libs/user/profile/core/facade/user-email.service.ts
+++ b/feature-libs/user/profile/core/facade/user-email.service.ts
@@ -1,5 +1,6 @@
 import { Injectable } from '@angular/core';
 import {
+  Command,
   CommandService,
   CommandStrategy,
   UserIdService,
@@ -17,10 +18,10 @@ export class UserEmailService implements UserEmailFacade {
     protected command: CommandService
   ) {}
 
-  protected updateCommand = this.command.create<{
+  protected updateCommand: Command<{
     password: string;
     newUid: string;
-  }>(
+  }> = this.command.create(
     (payload) =>
       this.userIdService
         .takeUserId(true)

--- a/feature-libs/user/profile/core/facade/user-password.service.ts
+++ b/feature-libs/user/profile/core/facade/user-password.service.ts
@@ -1,5 +1,5 @@
 import { Injectable } from '@angular/core';
-import { CommandService, UserIdService } from '@spartacus/core';
+import { Command, CommandService, UserIdService } from '@spartacus/core';
 import { Observable } from 'rxjs';
 import { switchMap, take } from 'rxjs/operators';
 import { UserPasswordFacade } from '@spartacus/user/profile/root';
@@ -7,10 +7,10 @@ import { UserProfileConnector } from '../connectors/user-profile.connector';
 
 @Injectable()
 export class UserPasswordService implements UserPasswordFacade {
-  protected updateCommand = this.command.create<{
+  protected updateCommand: Command<{
     oldPassword: string;
     newPassword: string;
-  }>((payload) =>
+  }> = this.command.create((payload) =>
     this.userIdService.takeUserId(true).pipe(
       take(1),
       switchMap((uid) =>
@@ -23,16 +23,16 @@ export class UserPasswordService implements UserPasswordFacade {
     )
   );
 
-  protected resetCommand = this.command.create<{
+  protected resetCommand: Command<{
     token: string;
     password: string;
-  }>((payload) =>
+  }> = this.command.create((payload) =>
     this.userProfileConnector.resetPassword(payload.token, payload.password)
   );
 
-  protected requestForgotPasswordEmailCommand = this.command.create<{
+  protected requestForgotPasswordEmailCommand: Command<{
     email: string;
-  }>((payload) =>
+  }> = this.command.create((payload) =>
     this.userProfileConnector.requestForgotPasswordEmail(payload.email)
   );
 

--- a/feature-libs/user/profile/core/facade/user-profile.service.ts
+++ b/feature-libs/user/profile/core/facade/user-profile.service.ts
@@ -1,10 +1,12 @@
 import { Injectable } from '@angular/core';
 import {
   AuthService,
+  Command,
   CommandService,
   CommandStrategy,
   EventService,
   LanguageSetEvent,
+  Query,
   QueryService,
   UserIdService,
 } from '@spartacus/core';
@@ -20,7 +22,7 @@ import { map, switchMap, tap } from 'rxjs/operators';
 
 @Injectable()
 export class UserProfileService implements UserProfileFacade {
-  protected updateCommand = this.command.create<{ details: User }>(
+  protected updateCommand: Command<{ details: User }> = this.command.create(
     (payload) =>
       this.userIdService.takeUserId(true).pipe(
         switchMap((uid) =>
@@ -39,7 +41,7 @@ export class UserProfileService implements UserProfileFacade {
     }
   );
 
-  protected closeCommand = this.command.create(() =>
+  protected closeCommand: Command = this.command.create(() =>
     this.userIdService
       .takeUserId(true)
       .pipe(
@@ -51,7 +53,7 @@ export class UserProfileService implements UserProfileFacade {
       )
   );
 
-  protected titleQuery = this.query.create(
+  protected titleQuery: Query<Title[]> = this.query.create(
     () => this.userProfileConnector.getTitles(),
     {
       reloadOn: [LanguageSetEvent],

--- a/feature-libs/user/profile/core/facade/user-register.service.ts
+++ b/feature-libs/user/profile/core/facade/user-register.service.ts
@@ -1,5 +1,10 @@
 import { Injectable } from '@angular/core';
-import { AuthService, CommandService, UserActions } from '@spartacus/core';
+import {
+  AuthService,
+  Command,
+  CommandService,
+  UserActions,
+} from '@spartacus/core';
 import { User } from '@spartacus/user/account/root';
 import { Observable } from 'rxjs';
 import {
@@ -14,25 +19,27 @@ import { Store } from '@ngrx/store';
 
 @Injectable()
 export class UserRegisterService implements UserRegisterFacade {
-  protected registerCommand = this.command.create<{ user: UserSignUp }, User>(
-    ({ user }) =>
-      this.userConnector.register(user).pipe(
-        tap(() => {
-          // this is a compatibility mechanism only, to make anonymous consents
-          // management work properly in transitional period (when we move logic
-          // to separate libraries)
-          this.store.dispatch(new UserActions.RegisterUserSuccess());
-        })
-      )
+  protected registerCommand: Command<
+    { user: UserSignUp },
+    User
+  > = this.command.create(({ user }) =>
+    this.userConnector.register(user).pipe(
+      tap(() => {
+        // this is a compatibility mechanism only, to make anonymous consents
+        // management work properly in transitional period (when we move logic
+        // to separate libraries)
+        this.store.dispatch(new UserActions.RegisterUserSuccess());
+      })
+    )
   );
 
-  protected registerGuestCommand = this.command.create<
+  protected registerGuestCommand: Command<
     {
       guid: string;
       password: string;
     },
     User
-  >((payload) =>
+  > = this.command.create((payload) =>
     this.userConnector.registerGuest(payload.guid, payload.password).pipe(
       tap((user) => {
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion

